api/metrics/v1/http.go

```diff
@@ -34,18 +34,28 @@ const (
 	ReceiveRoute     = "/api/v1/receive"
 	RulesRoute       = "/api/v1/rules"
 	RulesRawRoute    = "/api/v1/rules/raw"
+
+	AlertmanagerAlertsRoute   = "/api/v2/alerts"
```

Comment:
I'm not sure about the naming of this endpoint. Thanos Query also has an `/api/v1/alerts` API I believe, which makes such an endpoint kind of confusing for user (v1 to query, v2 to am).

Maybe we can attach an `am` prefix? So `am/api/v2/alerts`?

Reply:
yes I think that makes sense